crates/volta-core/src/tool/package/fetch.rs

```diff
@@ -2,10 +2,11 @@
 
 use std::fs::{rename, write, File};
 use std::io::{Read, Seek, SeekFrom};
-use std::path::{Path, PathBuf};
+use std::path::Path;
 
+use super::super::registry::find_unpack_dir;
```

Comment:
Nit: once I hit `super::super::` (really, anything beyond one `super`) I tend to reach for `crate::` instead because I find that easier to follow. Not a thing I think you *need* to change, by any means, just my own feeling.

Reply:
Yeah, it's pretty close between `super::super::` and `crate::tool::`. Not a big deal either way, I think my personal cutoff is at 3 😀